extmod/asyncio/core.py

```diff
@@ -77,13 +80,13 @@ def _enqueue(self, s, idx):
             entry = [None, None, s]
             entry[idx] = cur_task
             self.map[id(s)] = entry
-            self.poller.register(s, select.POLLIN if idx == 0 else select.POLLOUT)
+            self.poller.register(s, POLLIN if idx == 0 else POLLOUT)
         else:
             sm = self.map[id(s)]
             assert sm[idx] is None
             assert sm[1 - idx] is not None
             sm[idx] = cur_task
-            self.poller.modify(s, select.POLLIN | select.POLLOUT)
+            self.poller.modify(s, POLLIN | POLLOUT)
```

Comment:
any measurable benefit to having `POLLANY = POLLIN | POLLOUT` to avoid the calculation here?

Reply:
If they are declared `const` the calculation is done at compile time.